src/components/table-header.js

```diff
@@ -106,7 +106,7 @@ const TableHeader = ({
             <Grid item xs={3}>
                 <TaskStatusChip
                     data-test="timestamp-status"
-                    taskStatus={taskStatus}
+                    taskstatus={taskStatus}
```

Comment:
With task**S**tatus react is looking for a property task-status in the component. It is the way react works. 
So in order to remove the error from the console I change the name of the parameter. 
For simplicity I choose this correction instead of trying with task-status. It is easier to follow the parameter in the component and in his call.